srcs/django_code/front/js/game/canvas.js

```diff
@@ -1,4 +1,4 @@
-var canvas = document.getElementById("pong_canvas")
+var canvas = document.querySelector("#pong_canvas");
```

Comment:
```suggestion
let canvas = document.querySelector("#pong_canvas");
```
ou
```suggestion
const canvas = document.querySelector("#pong_canvas");
```
Si `canvas` n'est pas réassigner quelque part

Reply:
fixed